backend/Models/Response/SightingResponse.cs

```diff
@@ -9,4 +9,5 @@ public class SightingResponse
     public string PhotoUrl { get; set; }
     public string? Description { get; set; }
     public DateTime DateTime { get; set; }
+    public object UserId1 { get; }
```

Comment:
What's this been added for?

Reply:
looks like an unwanted quick fix on Vs code that generated this line 
should be removed

Reply:
removed